src/codecoverage/bot/code_coverage_bot/codecov.py

```diff
@@ -166,7 +138,7 @@ def go(self):
         else:
             logger.info('Generating suite reports')
             os.makedirs(self.ccov_reports_dir, exist_ok=True)
-            self.generate_suite_reports()
+            suite_reports.generate_suite_reports(self.suites, self.artifactsHandler, self.ccov_reports_dir, self.repo_dir)
```

Comment:
The function in suite_reports is not called `generate_suite_reports`, but `generate`.